modules/rhel-compute-requirements.adoc

```diff
@@ -51,6 +51,10 @@ be configured according to its
 link:https://vmware.github.io/vsphere-storage-for-kubernetes/documentation/index.html[storage guidelines]
 and the `disk.enableUUID=true` attribute must be set.
 
+* Each system should be able to access the cluster's API
+endpoints using a DNS resolvable hostname. Any network security access control in place should allow the system access to the
```

Comment:
Is the first sentence really:

"Each system must be able to access the cluster's API endpoints WHILE using a DNS resolvable hostname. "

or

"Each system must be able to access the cluster's API endpoints BY using a DNS resolvable hostname."

Reply:
Second one will fix